scripts/autonomous/autonomous_runner.py

```
@@ -0,0 +1,391 @@
+                f"**Success Rate:** {monitor['success_rate']}\n"
+                f"**Total Cost:** {monitor['total_cost']}\n"
+                f"**Total Time:** {monitor['elapsed_time_hours']}\n\n"
+                f"**Budget Remaining:**\n"
+                f"  - Cost: {monitor['cost_remaining']}\n"
+                f"  - Time: {monitor['time_remaining_hours']}\n"
+            )
+
+            # TODO: Send via Telegram bot
+            logger.info(f"\n[Telegram notification would be sent]:\n{message}")
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)